### PR TITLE
[Service Offloading] Support Multitouch on Android

### PR DESCRIPTION
**Pull request by Chikdolman**
Since we only support mouse event, FHUB and kiosk are not supported.
Therefore we need to support multitouch.
We enable multi-touch with this patch.

### CODE REVIEW

File: third_party/blink/renderer/modules/input_control/InputControl.java [hunk 1]
```diff
@@ -58,7 +70,50 @@ private static InputControl CreateInputControl() {
...
+    }
+
+    @CalledByNative
+    public void SendTouchInput(String type, String json) {
+        EnterpriseDeviceManager edm = EnterpriseDeviceManager.getInstance(
+          ContextUtils.getApplicationContext());
+        RemoteInjection remoteInjection = edm.getRemoteInjection();
+        int action = -1;
+
+        if (type.equals("touchstart") && mNumTouch < 1) {
```
Comment (Sunwoo-Nam) [inline third_party/blink/renderer/modules/input_control/InputControl.java]: How about this?
if (type.equals("touchstart")) {
  if (mNumTouch < 1) {
    ....
  } else {
    .....
  }
} else if (type.equals("touchend")) {
  ....
} else if (type.equals("touchmove")) {
  ....
}

Comment (Chikdolman) [reply third_party/blink/renderer/modules/input_control/InputControl.java]: Thank you 👍 applied

**Review by Sunwoo-Nam — APPROVED** [review]
Please merge if the behavior is verified.

**Review by kenxiniz — APPROVED** [review]
lgtm

**Pull request merged**